packages/webpack-cli/lib/utils/cli-flags.js

```diff
@@ -92,7 +92,6 @@ const core = [
         usage: '--color',
         type: Boolean,
         negative: true,
-        defaultValue: true,
         description: 'Enable/Disable colors on console',
```

Comment:
the `color` needs some more fixes, I'm removing it in
https://github.com/webpack/webpack-cli/pull/2042#discussion_r518730261